server/fleet/errors.go

```diff
@@ -294,6 +294,11 @@ func (e passwordResetRequiredError) StatusCode() int {
 // without having MDM correctly configured.
 type MDMNotConfiguredError struct{}
 
+// NewMDMNotConfiguredError returns a MDMNotConfiguredError.
+func NewMDMNotConfiguredError() *MDMNotConfiguredError {
+	return &MDMNotConfiguredError{}
+}
```

Comment:
I don't think it's worth adding a constructor function if it's just a literal struct creation? I think constructors should be if the caller can't initialize the struct to a usable state just via its fields.

Reply:
Sounds good. I was on the fence about this one. I'll remove it.